src/pipeline.ml

```diff
@@ -64,9 +64,9 @@ let install_compiler_df ~distro ~arch ~switch ?windows_port opam_image =
     | None ->
       (fun fmt -> run fmt), (fun fmt -> run fmt), "opam-depext", ["opam"; "exec"; "--"]
     | Some `Msvc ->
-      (fun fmt -> run_ocaml_env [bitness; "--ms=vs2019"] fmt),
-      (fun fmt -> run_ocaml_env [bitness; "--ms=vs2019"; "--no-opam"] fmt),
-       "depext", ["ocaml-env"; "exec"; bitness; "--ms=vs2019"; "--"]
+      (fun fmt -> run_ocaml_env [bitness; "--ms"] fmt),
```

Comment:
What is the significance of removing `vs2019` ? 
What version of Visual Studio does it default to using?

Reply:
I've updated the image to VS2022. Requiring VS2019 from `ocaml-env` fails, but using simply the `--ms` makes it look for any version of VS. As there is only one installed in the image, it'll default to VS2022 now. It's odd that ocaml-env finds VS2022 seemingly without any update on its part, whereas msvs-detect needed one. Anyway, it works this way and the Dockerfile produces a working opam switch, capable of building C stubs and OCaml code.